### DISCUSSION TITLE
Found a fix for interrogator

### DISCUSSION THREAD

**Discussion by createperhaps** [TheLastBen/fast-stable-diffusion#1192] [General]
Hello,
I found out that the interrogator works if you run 

`pip install transformers==4.19.2`

in a cell above start stable diffusion. I haven't found any downsides to using 4.19 (it uninstalled 4.20), so I can just run it locally if I need it, but I haven't fully tested if this breaks anything else. Feel free to use. That is why I am hesitant to post this in an issue tab.

**Comment by TheLastBen** [comment]
https://github.com/TheLastBen/fast-stable-diffusion/issues/1056

I'm aware of it, I'm just making sure it won't break anything else.

**Comment by asdomedio** [comment]
a no .25.1 ned
